src/cmd/commit.rs

```diff
@@ -294,7 +365,26 @@ impl Command for CommitCommand {
         }
         .wizard(&config, parser, self.interactive)?;
 
-        self.commit(msg)?;
+        std::fs::write(commit_editmsg_path, &msg)?;
+        if !is_git_editor {
+            self.commit_msg_and_remove_file(&msg, commit_editmsg_path)?;
+        }
+
         Ok(())
     }
 }
+
+fn config_types_to_conventional(types: &[Type]) -> Vec<crate::conventional::Type> {
+    types
+        .iter()
+        .map(|ty| ty.r#type.as_str())
+        .map(crate::conventional::Type::from)
+        .collect()
+}
+
+fn get_default_commit_msg_path() -> PathBuf {
+    git2::Repository::open_from_env()
+        .unwrap()
```

Comment:
This unwrap does make for a slightly ugly error message if you run convco in a non-git dir:

```
could not find repository at '.'; class=Repository (6); code=NotFound (-3)
thread 'main' panicked at src/cmd/commit.rs:387:10:
called `Result::unwrap()` on an `Err` value: Error { code: -3, klass: 6, message: "could not find repository at '.'" }
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Perhaps gracefully handling the error would be valuable.